pkg/engine/virtualmachines.go

```diff
@@ -229,6 +245,58 @@ func CreateMasterVM(cs *api.ContainerService) VirtualMachineARM {
 	}
 }
 
+func createEtcdDisk(cs *api.ContainerService) DiskARM {
+	var ssdType compute.DiskStorageAccountTypes
+	if cs.Properties.MasterProfile != nil {
+		switch cs.Properties.MasterProfile.DataDiskSSDType {
+		case api.Ultra:
+			ssdType = compute.UltraSSDLRS
+		case api.Premium:
+			ssdType = compute.PremiumLRS
+		case api.Standard:
+			ssdType = compute.StandardLRS
```

Comment:
Standard SSD should actually be `StandardSSDLRS `. `StandardLRS ` is standard HDD:

https://github.com/Azure/aks-engine/blob/b3c4f9eb7e0bd91f7b5977f8534ed96c21a1355c/vendor/github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-12-01/compute/models.go#L378